Makefile

```diff
@@ -10,27 +10,24 @@ GOBUILD_VERSION_ARGS_WITH_SYMS := -ldflags "-X $(VERSION_VAR)=$(REPO_VERSION) -X
 BINARY_NAME := gostatsd
 IMAGE_NAME := atlassianlabs/$(BINARY_NAME)
 ARCH ?= $$(uname -s | tr A-Z a-z)
-GOVERSION := 1.13  # Keep in sync with .travis.yml
+GOVERSION := 1.13.6  # Keep in sync with .travis.yml
 GP := /gopath
 MAIN_PKG := github.com/atlassian/gostatsd/cmd/gostatsd
 CLUSTER_PKG := github.com/atlassian/gostatsd/cmd/cluster
 PROTOBUF_VERSION ?= 3.6.1
 
 setup: setup-ci
-	go get github.com/githubnemo/CompileDaemon
-	go get github.com/jstemmer/go-junit-report
-	go get golang.org/x/tools/cmd/goimports
+	go install github.com/githubnemo/CompileDaemon \
+		github.com/jstemmer/go-junit-report \
+		golang.org/x/tools/cmd/goimports
```

Comment:
`go get` is a command to get a specific version of a dependency. If no version is specified it tries to get the `latest`. `go install` just installs the version of the binary that is tracked in `go.mod`/`go.sum`.

Reply:
In addition here we pull all dependencies using 1 command - can do more concurrent downloading and should be faster.